### ISSUE TITLE
Support new way to disable rules in ktlint 0.48

### ISSUE THREAD

**Issue by bcmedeiros** [diffplug/spotless#1444]
- [ 6.12.1 ] spotless version

Currently, using the `ktlint_disabled_rules` causes many warnings to be printed in the console:
```
Property 'ktlint_disabled_rules' is deprecated: Update all your all '.editorconfig' files. See [link below]
```

https://pinterest.github.io/ktlint/faq/#why-is-editorconfig-property-disabled_rules-deprecated-and-how-do-i-resolve-this

Following that, if I configure Spotless as below, the config is ignored and rules are still enabled.
```
configure<com.diffplug.gradle.spotless.SpotlessExtension> {
    kotlin {
        ktlint("0.48.0")
            .editorConfigOverride(
                mapOf(
                    "ktlint_standard_filename" to "disabled",
                ),
            )
    }
}
```

**Comment by bcmedeiros** [comment]
This is really bad with Spotless since we invoke ktlint for each file, so tons of warnings.

**Comment by twarner-sofi** [comment]
I get `java.lang.IllegalArgumentException: Can not create an EditorConfigOverride without properties. Use 'emptyEditorConfigOverride' instead.` when trying to use the new property format.

spotless 6.12.1

**Comment by nedtwigg** [comment]
Fix published in `plugin-gradle 6.13.0` and `plugin-maven 2.30.0`.
